prboom2/src/SDL/i_system.c

```diff
@@ -236,7 +236,12 @@ void I_SwitchToWindow(HWND hwnd)
   }
 }
 
-const char *I_DoomExeDir(void)
+const char *I_DoomDir(void)
```

Comment:
Shall we rather call this `I_ConfigDir`?

Reply:
Hm, maybe `I_DataDir`? It's maybe too generic, plus confusing since dsda-doom has a "data dir" as well that's one level below that. Any other ideas?

Reply:
I like `I_ConfigDir`.
Its pretty clear what it represents, so no one will be confused.